### PR TITLE
dev-util/conan: add 2.4.1

### PR DESCRIPTION
**Pull request by dpronin**
Closes: https://bugs.gentoo.org/934035

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by dpronin** [comment]
@winterheart will we merge it?

File: dev-util/conan/conan-2.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	>=dev-python/requests-2.25[${PYTHON_USEDEP}]
+	>=dev-python/urllib3-1.26.6[${PYTHON_USEDEP}]
+	>=dev-python/colorama-0.4.3[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-6.0[${PYTHON_USEDEP}]
+	>=dev-python/patch-ng-1.17.4[${PYTHON_USEDEP}]
+	>=dev-python/fasteners-0.15.0[${PYTHON_USEDEP}]
+	>=dev-python/jinja-3.0[${PYTHON_USEDEP}]
+	>=dev-python/python-dateutil-2.8.0[${PYTHON_USEDEP}]
+	>=dev-python/distro-1.4.0[${PYTHON_USEDEP}]
+"
```
Comment (winterheart) [inline dev-util/conan/conan-2.4.1.ebuild:30]: Please reorder dependencies alphabetically.

Dependencies from https://github.com/conan-io/conan/blob/release/2.4/conans/requirements_server.txt are missing:

```
bottle>=0.12.8, < 0.13
pluginbase>=0.5
PyJWT>=2.4.0, <3.0.0
```

**Comment by dpronin** [comment]
@winterheart check it out

**Comment by winterheart** [comment]
OK from me.

**Comment by dpronin** [comment]
> OK from me.

And who is able to merge?

**Comment by dpronin** [comment]
@thesamesam would you merge it?

**Review by arthurzam — APPROVED** [review]

**Pull request closed without merging**